.github/auto_request_review.yml

```diff
@@ -0,0 +1,62 @@
+reviewers:
+  # The default reviewers
+  defaults:
+    - code-owners # group
+
+  # Please note reviewers do not need to be restricted to people
+  # who have write access. TLs and Emeriti Chairs have write
+  # access and Co-chairs have admin access
+  # Once a member who does not have "write/push" access, finishes
+  # their review, they should assign it to one of the code-owners
+  # mentioned below for approval
+  groups:
+    code-owners:
+      - ultrasaurus
+      - dshaw
+      - pragashj
+      - lumjjb
+      - TheFoxAtWork
+      - JustinCappos
+      - achetal01
+      - ashutosh-narkar
+      - anvega
+      - PushkarJ
+      - mnm678
+      - mlieberman85
+      - ragashreeshekar
+      - sublimino
+    supply-chain-security-reviewers:
+      - mnm678
+      - mlieberman85
+      - anvega
+      - lumjjb
+      - SantiagoTorres
+    whitepaper-reviewers:
+      - PushkarJ
+      - ragashreeshekar
+      - TheFoxAtWork
+      - achetal01
+
+files:
+  # Keys are glob expressions.
+  # You can assign groups defined above as well as GitHub usernames.
+  'security-whitepaper/**':
+    - whitepaper-reviewers # group
+  'landscape/**':
+    - whitepaper-reviewers #group
+  'security-lexicon/**':
+    - whitepaper-reviewers #group
+  'supply-chain-security/**':
+    - supply-chain-security-reviewers #group
+
+options:
```

Comment:
Can you please add a note to describe options? Also if there are any links for additional options.

Reply:
done